### DISCUSSION TITLE
Worked great for Field Day 2024

### DISCUSSION THREAD

**Discussion by rufusrizzo** [n1kdo/n1mm_view#61] [General]
I just wanted to sat Thanks!  This worked great and was a huge hit at Field day 2024 for the VARA and MARA clubs!   We were w4xd if you heard us.

We had some people making more QSOs to move up in the list, and many people like the quick stats.  I'm putting the info on little displays so they can see if off the computer screen.

**Comment by n1kdo** [comment]
Thanks for your comments.  Several people have helped make this happen!
